lxd/cluster/recover.go

```diff
@@ -136,6 +136,39 @@ func updateLocalAddress(database *db.Node, address string) error {
 	return nil
 }
 
+// Create patch file for global nodes database.
```

Comment:
I think it would be useful to expand this comment to explain what the patch does.